include/Model/Languages.php

```diff
@@ -1025,11 +1055,15 @@ protected function update_translations( $old_slug, $new_slug = '' ): void {
 					$dt['tt']
 				)
 			);
+			if ( false === $result ) {
+				/* translators: %s is the error message from the database */
+				$errors->add( 'pll_delete_term_taxonomy', __( 'Impossible to delete the term taxonomy, database error: %s.', 'polylang' ), $wpdb->last_error );
```

Comment:
Users don't know `term_taxonomy`:
```suggestion
				$errors->add( 'pll_delete_term_taxonomy', __( 'Could not delete translation groups.', 'polylang' ), $wpdb->last_error );
```